internal/integration/api/extensions_qemu.go

```diff
@@ -68,14 +68,14 @@ func (suite *ExtensionsSuiteQEMU) TearDownTest() {
 // TestExtensionsExpectedPaths verifies expected paths are present.
 func (suite *ExtensionsSuiteQEMU) TestExtensionsExpectedPaths() {
 	expectedPaths := []string{
-		"/lib/firmware/amdgpu",
-		"/lib/firmware/amd-ucode",
-		"/lib/firmware/bnx2x",
-		"/lib/firmware/cxgb3",
-		"/lib/firmware/cxgb4/configs",
-		"/lib/firmware/i915",
-		"/lib/firmware/intel/ice/ddp",
-		"/lib/firmware/intel-ucode",
+		"/usr/lib/firmware/amdgpu",
```

Comment:
I think we can keep existing paths, this also ensures that the symlinks work

Reply:
should do it